### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by MoniS10**
Added some other unneeded files

### CODE REVIEW

**Comment by leon332157** [comment]
thank you

File: .gitignore [hunk 1]
```diff
@@ -13,6 +13,9 @@ pre-dist
 
 # npm
 node_modules
+package-lock.json
+yarn.lock
+yarn-error.log
```
Comment (eankeen) [inline .gitignore]: covered by `*.log`

**Comment by eankeen** [comment]
hey! thanks for your contribution! sorry if we merged the changes a bit prematurely, but i dont think that these are the changes we are looking for. lockfiles gotta be committed to vcs.

thanks for you interest, maybe you can help with issue #15 or #13 - i made some changes to the project a while back which made the development experience a bit more snappy :P

**Comment by MoniS10** [comment]
sure thing. I'll take a look 👍

**Comment by leon332157** [comment]
Uh, what is going on.

**Comment by BetiDev** [comment]
Hullo!

**Comment by leon332157** [comment]
@BetiDev Hi, if you want to discuss with us for the app, you can open a new issue or just talk in discord because this is a close PR. If you need help setting up github for contributing. Me and @eankeen can help you :)

**Pull request merged**